src/theme/components/heading.ts

```diff
@@ -1,39 +1,46 @@
+const margins = {
+  marginTop: 0,
+  marginLeft: 0,
+  marginRight: 0,
+  marginBottom: 2, // var(--space-xs)
```

Comment:
After seeing how this responded, this should be bumped up to `marginBottom: 4` to align with `--space-s`.